pkg/controllers/restore/controller.go

```diff
@@ -101,6 +107,12 @@ func Register(
 		restmapper:              restmapper,
 		defaultBackupMountPath:  defaultLocalBackupLocation,
 		defaultS3BackupLocation: defaultS3,
+		kubernetesLeaseClient:   leaseClient,
+	}
+
+	lease, err := leaseClient.Get(ctx, leaseName, k8sv1.GetOptions{})
+	if err == nil && lease != nil {
+		leaseClient.Delete(ctx, leaseName, k8sv1.DeleteOptions{})
```

Comment:
I think this could cause issues in the future if we have more than one running but should be fine for now.

Reply:
made an issue for 2.5.x to track it